offchain-modules/packages/x/src/core.ts

```diff
@@ -62,18 +61,30 @@ export class ForceBridgeCore {
     return ForceBridgeCore._ckbIndexer;
   }
 
+  static get keystore(): KeyStore {
+    asserts(ForceBridgeCore._keystore, 'ForceBridgeCore is not init yet');
+    return ForceBridgeCore._keystore;
+  }
+
+  /**
+   * @deprecated migrate to {@link bootstrap}
+   */
+  constructor() {
+    // TODO make constructor to be private
+  }
+
   async init(config: Config): Promise<ForceBridgeCore> {
     ForceBridgeCore.initiated = true;
     ForceBridgeCore._config = config;
     ForceBridgeCore._ckb = new CKB(config.ckb.ckbRpcUrl);
     ForceBridgeCore._ckbIndexer = new CkbIndexer(config.ckb.ckbRpcUrl, config.ckb.ckbIndexerUrl);
-    ForceBridgeCore._keystore = keystore;
 
     // init log
     initLog(config.common.log);
 
     // decrypt private key
     const keystore = bootstrapKeyStore(config.common.keystorePath);
+    ForceBridgeCore._keystore = keystore;
     // ForceBridgeCore._keystore = keystore;
     ForceBridgeCore.config.ckb.fromPrivateKey = keystore.getDecryptedByKeyID(config.ckb.fromPrivateKey);
```

Comment:
the `ckb` private key is not required for multisig-server, maybe we need check if `ckb` is in the keystore first

Reply:
we may remove the `multiSignKeys` and use `ckb` and `eth` for all roles, just left it to next PR.